server/models/databaseModels.js

```diff
@@ -0,0 +1,7 @@
+const users = "DROP TABLE IF EXISTS users CASCADE; DROP TYPE IF EXISTS user_status; CREATE TYPE user_status AS ENUM ('user', 'admin'); CREATE TABLE users(id SERIAL PRIMARY KEY,username VARCHAR (20) NOT NULL UNIQUE,fullname VARCHAR (50) NOT NULL,email VARCHAR (50) NOT NULL UNIQUE,password VARCHAR (240) NOT NULL,role user_status default 'user',created_at timestamp (0) without time zone default now());";
```

Comment:
This naming is not descriptive enough. `createUsersTableQuery` is better for this.
Make this correction and improve on all others